### ISSUE TITLE
Would it be possible to do this on iOS

### ISSUE THREAD

**Issue by stossy11** [devos50/qemu#125]
The UTM team has ported QEMU to iOS so can you run this on iOS.

**Comment by devos50** [comment]
Yup, that would technically be possible I guess, but probably the touch/keyboard inputs have to be reworked 👍
